src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('span.population')];
+
+const populationArr = elements.map((element) => {
+  return Number(element.innerText.split(',').join(''));
+});
```

Comment:
Here, you are creating an array of populations using the `map` method. This is a good use of the `map` method, but it's worth noting that the `map` method creates a new array. If the original array is large, this can be a performance issue. In this case, it's probably not a problem, but it's something to keep in mind.